.github/workflows/trivy.yml

```diff
@@ -0,0 +1,22 @@
+name: Trivy scan
+on: [pull_request]
+
+jobs:
+  appsec-trivy:
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: /tmp/jar-cache
+          key: ${{ runner.os }}-${{ github.ref }}
+
+      - name: Build jar
```

Comment:
So, we've got two options. We can build the artifact properly, like this, or we _could_ just touch the file and not bother. Thoughts?

Reply:
Nice out-the-box thinking here! Agreed, touch is a simpler workaround.